railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -309,6 +309,7 @@ class AppGenerator < AppBase
           api: [
             :skip_asset_pipeline,
             :skip_javascript,
+            :skip_css
```

Comment:
There is currently no `--skip-css` option, so I don't think this is doing anything